provider/terraform/resource_override.rb

```diff
@@ -19,13 +19,40 @@ class Terraform < Provider::AbstractCore
     # Collection of properties allowed in the ResourceOverride section for
     # Terraform. All properties should be `attr_reader :<property>`
     module OverrideProperties
+      # The Terraform resource id format used when calling #setId(...).
+      # For instance, `{{name}}` means the id will be the resource name.
+      attr_reader :id_format
+
+      attr_reader :documentation
+    end
+
+    # Terraform specific documentation additions
+    class Documentation < Api::Object
```

Comment:
I suggest making examples part of the OverrideProperties. As of now there's not much grouping of documentation that warrants the "extra hassle" of defining another !ruby/object in the terraform.yaml file.

Reply:
Done